_data/simple-icons.json

```diff
@@ -4565,6 +4565,11 @@
             "source": "https://commons.wikimedia.org/wiki/File:GTK_logo.svg",
             "guidelines": "https://foundation.gnome.org/logo-and-trademarks/"
         },
+        {
+            "title": "Guacamole",
+            "hex": "304730",
+            "source": "https://github.com/apache/guacamole-website/blob/7b832798bb43f9047d0725b10604e09fd3245cbd/images/guac-logo.png"
```

Comment:
Official SVG source is available at [this link](https://www.apache.org/logos/#guacamole) which can be used for this.